packages/registrar_client/README.md

```diff
@@ -0,0 +1,32 @@
+# Registrar Client
+
+This package provides a client for interacting with the TFGrid v4 Node Registrar.
+
+## Getting Started
+
+Set the `REGISTRAR_URL` in your environment variables to point to the TFGrid v4 Node Registrar.
```

Comment:
No, that should be passed to the client e.g


Client(url=, mnemonic=.., ... etc)